be/src/formats/parquet/file_reader.cpp

```diff
@@ -82,21 +82,20 @@ Status FileReader::_parse_footer() {
     {
         SCOPED_RAW_TIMER(&_scanner_ctx->stats->footer_read_ns);
         RETURN_IF_ERROR(_file->read_at_fully(_file_size - footer_read_size, footer_buffer.data(), footer_read_size));
-        _scanner_ctx->stats->request_bytes_read += footer_read_size;
-        _scanner_ctx->stats->request_bytes_read_uncompressed += footer_read_size;
     }
 
     ASSIGN_OR_RETURN(uint32_t metadata_length, _parse_metadata_length(footer_buffer));
 
+    _scanner_ctx->stats->request_bytes_read += metadata_length;
```

Comment:
Put code below line::100. Because if line::98 failed, we don't need to plus `request_bytes_read`.